sources/programs/quine/rust/how-to-run-the-solution.md

```diff
@@ -0,0 +1,10 @@
+If you want to run this program, you'll first need to download a
+copy of the 
+[Rust compiler](https://www.rust-lang.org/tools/install),
+and follow the installation instructions. From there, open a terminal, and
+run this command:
+
+```
+rustc quine.eu
```

Comment:
Is the extension meant to be .eu?

Reply:
Oops, you caught me! I copy this file from Euphoria. I'll fix that